packages/integrations/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@sentry/integrations",
+  "version": "4.6.1",
```

You have to make sure the version use the latest `master` so `4.6.3` otherwise it installs old versions. Then the tests should also succeed. 
```suggestion
  "version": "4.6.3",
```